src/components/Form/Resolver.js

```diff
@@ -0,0 +1,17 @@
+// Assumes json has been parsed into a JavaScript object literal
+const mapJsonToFields = (json, componentsLookup, validatorsLookup) => {
```

Comment:
@alexthecarp There is really no meaningful change from the last version you saw, but, this codifies what was _in my head_ in terms of having a "Resolver" that maps somewhat arbitrary json, component tables, and validator tables, but in a way that should allow for various per use case setups. The team can set these up pretty much however it makes sense for whatever form they need to support.